### PR TITLE
kernel: bump 5.10 to 5.10.175

### PR DESCRIPTION
**Pull request by graysky2**
Manually rebased:
        backport-5.10/611-v5.12-net-ethernet-mediatek-support-setting-MTU.patch

Removed upstreamed:
        bcm47xx/patches-5.10/170-bgmac-fix-initial-chip-reset-to-support-BCM5358.patch[1]

All other patches automatically rebased.

1. https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/?h=v5.10.175&id=cbf11ff3708ff163387da924f80a47ce7c721e9b

### CODE REVIEW

**Comment by graysky2** [comment]
Not sure how to fix the error for lantiq

File: target/linux/ramips/patches-5.10/810-uvc-add-iPassion-iP2970-support.patch [hunk 1]
```diff
@@ -225,18 +225,16 @@ Signed-off-by: John Crispin <blogic@openwrt.org>
...
 +++ b/drivers/media/usb/uvc/uvcvideo.h
-@@ -203,7 +203,9 @@
- #define UVC_QUIRK_RESTORE_CTRLS_ON_INIT	0x00000400
+@@ -204,6 +204,8 @@
  #define UVC_QUIRK_FORCE_Y8		0x00000800
  #define UVC_QUIRK_FORCE_BPP		0x00001000
--
+ #define UVC_QUIRK_WAKE_AUTOSUSPEND	0x00002000
 +#define UVC_QUIRK_MOTION		0x00001000
 +#define UVC_QUIRK_SINGLE_ISO		0x00002000
```
Comment (hauke) [inline target/linux/ramips/patches-5.10/810-uvc-add-iPassion-iP2970-support.patch:233]: Please use new numbers for these new defines.

**Comment by hauke** [comment]
You can just remove `unsigned long physical_memsize = 0L;` from the lantiq patch. 
`physical_memsize` was previously used to set in `mttgpr(7, physical_memsize);` and this was changed to hard code 0, see:
https://github.com/gregkh/linux/commit/bccccd43a06dfea5f5a8295a0cb384d74b90a9b3

File: target/linux/ramips/patches-5.10/810-uvc-add-iPassion-iP2970-support.patch (hunk 1 above)
Comment (graysky2) [reply target/linux/ramips/patches-5.10/810-uvc-add-iPassion-iP2970-support.patch:233]: Done will push soon

File: target/linux/x86/config-5.10 [hunk 2]
```diff
@@ -218,7 +218,7 @@ CONFIG_MEMFD_CREATE=y
 CONFIG_MICROCODE=y
 CONFIG_MICROCODE_AMD=y
 CONFIG_MICROCODE_INTEL=y
-CONFIG_MICROCODE_OLD_INTERFACE=y
+# CONFIG_MICROCODE_LATE_LOADING is not set
```
Comment (hauke) [inline target/linux/x86/config-5.10:221]: Please activate it here too.

Comment (graysky2) [reply target/linux/x86/config-5.10]: done

**Pull request merged**